Automattic/jetpack
# VideoPress: Add original core/video attributes to extended attributes definition for v5

Fixes #30571

## Proposed changes:

* Adds the original `core/video` attributes (this issue particularly needs the `tracks` attribute) to the blocks attributes so the saved content matches the save function

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Do NOT install Jetpack
* Go to the block editor and add a `core/video` block
* Upload subtitles to the videos tracks and save the post
* Hard refresh
* Check that the block is rendered as normal
* Install and activate Jetpack
* Go back to the post in the block editor
* Hard refresh
* Check that the block is rendered as normal, without invalid content
* Check that the subtitles work as expected

Before | After
-|-
![2023-05-15_16-39-59](https://github.com/Automattic/jetpack/assets/8486249/5b4e222f-7197-4d90-a841-e3391f119f14) | ![2023-05-15_16-40-18](https://github.com/Automattic/jetpack/assets/8486249/f9268c28-83f4-4173-99c3-7f752e60aecf)
